### PR TITLE
Manezihno's Hunter Sprites

### PR DESCRIPTION
**Pull request by Surrealaser**
:cl: by Manezihno and Surrealistik
tweak: New more stealth friendly sprites for the Hunter.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/death.dm [hunk 1]
```diff
@@ -126,6 +128,8 @@
 		to_flick = "gibbed-a-runner"
 	else if(isxenolarva(src))
 		to_flick = "larva_gib"
+	else if(isxenohunter(src))
```
Comment (DominikPanic) [inline code/modules/mob/living/carbon/xenomorph/death.dm]: Unacceptable

File: code/modules/mob/living/carbon/xenomorph/death.dm [hunk 2]
```diff
@@ -111,6 +111,8 @@
 		remains.icon_state = "gibbed-a-corpse-runner"
 	else if(isxenolarva(src))
 		remains.icon_state = "larva_gib_corpse"
+	else if(isxenohunter(src))
```
Comment (DominikPanic) [inline code/modules/mob/living/carbon/xenomorph/death.dm]: Same

Comment (Rohesie) [reply code/modules/mob/living/carbon/xenomorph/death.dm]: Just do redefinitions.
`/mob/living/carbon/Xenomorph/Hunter/gib()` and so on.

Comment (Surrealaser) [reply code/modules/mob/living/carbon/xenomorph/death.dm]: This is done

File: code/modules/mob/living/carbon/xenomorph/death.dm [hunk 3]
```diff
@@ -118,6 +118,17 @@
...
+
+	var/obj/effect/decal/remains/xeno/remains = new(get_turf(src))
+	remains.icon = icon
+	remains.pixel_x = pixel_x //For 2x2.
+
+	remains.icon_state = "Hunter Gibs"
+
+	check_blood_splash(35, BURN, 65, 2) //Some testing numbers. 35 burn, 65 chance.
+
+	..(1)
```
Comment (DominikPanic) [inline code/modules/mob/living/carbon/xenomorph/death.dm]:
```suggestion
	return ..()
```
There is nothing that takes a parameter in the parent procs.

Comment (Surrealaser) [reply code/modules/mob/living/carbon/xenomorph/death.dm]: I took it as I found it

**Pull request merged**